explorer/websockethandlers.go

```diff
@@ -21,6 +21,49 @@ import (
 // close an already closed connection.
 var ErrWsClosed = "use of closed network connection"
 
+func trimTxInfo(txs []*TxInfo) []*TrimmedTxInfo {
+	trimmedTxs := make([]*TrimmedTxInfo, 0, len(txs))
+	for _, tx := range txs {
+		voteValid := false
+		if tx.IsVote() {
+			voteValid = tx.VoteInfo.Validation.Validity
+		}
+		trimmedTx := &TrimmedTxInfo{
+			VinCount:  len(tx.Vin),
+			VoutCount: len(tx.Vout),
+			VoteValid: voteValid,
+			TxID:      tx.TxID,
+			Total:     tx.Total,
+			Fees:      tx.Fee.ToCoin(),
+			Coinbase:  tx.Coinbase,
+		}
+		trimmedTxs = append(trimmedTxs, trimmedTx)
+	}
+	return trimmedTxs
+}
+func trimBlockInfo(block *BlockInfo) *TrimmedBlockInfo {
```

Comment:
Please add a new line between these functions as per the style used in the rest of the project.